features/support/env.rb

```diff
@@ -4,6 +4,8 @@
 require 'cucumber/rspec/doubles'
 require 'capybara/poltergeist'
 require 'webmock/cucumber'
+require 'capybara-screenshot/cucumber'
```

Comment:
I thought we turned these off? I was accidentally committing them all the time, would be cool if we could have them turned on in specific envs instead of globally